src/currency/formatter/fn.js

```diff
@@ -0,0 +1,32 @@
+define([
+	"../../common/validate/parameter-presence",
+	"../../common/validate/parameter-type/number",
+	"../name-format"
+], function( validateParameterPresence, validateParameterTypeNumber, currencyNameFormat ) {
+
+return function( numberFormatter, pluralGenerator, properties ) {
+	var fn;
+
+	// Return formatter when style is "code" or "name".
+	if ( pluralGenerator && properties ) {
```

Comment:
This looks strange. Wouldn't it be better to split these two paths into two files and "include" just the one that's actually gonna be used?